target/stale/extract-native.dependencies

```diff
@@ -0,0 +1 @@
+([:dependencies ([org.clojure/clojure "1.5.1"] [org.clojure/tools.nrepl "0.2.10" :exclusions ([org.clojure/clojure])] [clojure-complete/clojure-complete "0.2.3" :exclusions ([org.clojure/clojure])] [lein-light-nrepl/lein-light-nrepl "0.1.3"] [lein-light-nrepl-instarepl/lein-light-nrepl-instarepl "0.3.1"])])
```

Comment:
don't commit this file either. Maybe worth adding `target` to your global .gitignore ?